src/khoj/routers/web_client.py

```diff
@@ -146,7 +154,7 @@ def content_config_page(request: Request, content_type: str):
         current_config = json.loads(current_config.json())
 
         return templates.TemplateResponse(
-            "content_type_input.html",
+            f"content_type_{content_type}_input.html",
```

Comment:
Please revert this and create a separate page specific to the new configuration (e.g., `content_type_website_input.html`) along with relevant new APIs for CRUD operations on the config.